dev/flutter/todo/submissions/Dishal Kanishka/todo_application/lib/screens/view_todo.dart

```diff
@@ -0,0 +1,156 @@
+                },
+                style: TextStyle(color: Theme.of(context).primaryColor),
+                decoration: InputDecoration(
+                  labelStyle: TextStyle(color: Theme.of(context).primaryColor),
+                  labelText: "Description",
+                  fillColor: Theme.of(context).primaryColor,
+                  border: OutlineInputBorder(
+                    borderRadius: BorderRadius.circular(10.0),
+                    borderSide:
+                        BorderSide(color: Theme.of(context).primaryColorLight),
+                  ),
+                  //fillColor: Colors.green
+                ),
+                controller: descriptionController,
+              ))),
+            ],
+          ),
+        ),
+      ),
+      bottomNavigationBar: Container(
+        height: 55.0,
+        child: BottomAppBar(
+          color: Theme.of(context).bottomAppBarTheme.color,
+          child: Row(
+            mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+            children: <Widget>[
+              InkWell(
+                  onTap: () {
+                    showDialog(
+                        context: context,
+                        builder: (ctx) => AlertDialog(
+                              title: Text("Alert"),
+                              content: Text(
+                                  "To Do ${todo.status ? 'not complete' : 'incomplete'}  "),
+                              actions: <Widget>[
+                                TextButton(
+                                  onPressed: () {
+                                    Navigator.of(ctx).pop();
+                                  },
+                                  child: Text("No"),
+                                ),
+                                TextButton(
+                                  onPressed: () {
+                                    setState(() {
+                                      todo.status = !todo.status;
+                                    });
+                                    Navigator.of(ctx).pop();
+                                  },
+                                  child: const Text("Yes"),
+                                )
+                              ],
+                            ));
+                  },
+                  child: Text(
+                    "${todo.status ? 'To Do Complete' : 'To Do Incomplete'} ",
+                    style: TextStyle(color: Theme.of(context).primaryColor),
+                  )),
+              const VerticalDivider(
+                color: Colors.white,
+              ),
+              IconButton(
+                icon: Icon(Icons.check_circle,
+                    color: Theme.of(context).primaryColor),
+                onPressed: () {
+                  Navigator.pop(context, todo);
+                },
+              )
+            ],
+          ),
+        ),
+      ),
+    );
+  }
+
+  Widget colorOverride(Widget child) {
```

Comment:
Widgets should be named with upper camel case by convention, and this should be somewhere around in a theme or a config file

Reply:
No I prevoiusly used it. Then I delete it